manager/state/store/memory.go

```diff
@@ -499,6 +527,7 @@ func (tx tx) changelistStoreActions() ([]api.StoreAction, error) {
 
 // lookup is an internal typed wrapper around memdb.
 func (tx readTx) lookup(table, index, id string) api.StoreObject {
+	defer metrics.StartTimer(lookupLatencyTimner)()
```

Comment:
I'm pretty sure the latency of a lookup is going to be less than the timer overhead. It's just an in-memory data structure lookup.

Maybe that's okay, but I feel this would add a lot of overhead, and the result wouldn't be especially interesting.

Reply:
Removed.